db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/Email.class/__methods__/servicetemplateprovisionrequest_denied.rb

```diff
@@ -0,0 +1,105 @@
+
+#
+# Description: This method is used to email the requester and approver that the service request has been denied
+#
+
+def send_mail(to, from, subject, body)
+  $evm.log(:info, "Sending email to #{to} from #{from} subject: #{subject}")
+  $evm.execute(:send_email, to, from, subject, body)
+end
+
+def requester
+  @miq_request.requester
+end
+
+def signature
+  $evm.object['signature']
+end
+
+def reason
+  @miq_request.reason
+end
+
+def approver_href(appliance)
+  body = " <a href='https://#{appliance}/miq_request/show/#{@miq_request.id}'</a>"
+  body
+end
+
+def approver_denied_text(requester_email, msg, reason)
+  body = "<br>A service request received from #{requester_email} was denied."
+  body += "<br><br>#{msg}."
+  body += "<br><br>Approvers notes: #{reason}"
+  body
+end
+
+def approver_text(appliance, msg, requester_email)
+  body = "Approver, "
+  body += approver_denied_text(requester_email, msg, reason)
+  body += "<br><br>For more information you can go to:"
+  body += approver_href(appliance)
+  body += "<br><br> Thank you,"
+  body += "<br> #{signature}"
+  body
+end
+
+def requester_email_address
+  owner_email = @miq_request.options.fetch(:owner_email, nil)
+  email = requester.email || owner_email || $evm.object['to_email_address']
+  $evm.log(:info, "Requester email: #{email}")
+  email
+end
+
+def email_approver(appliance, msg)
+  $evm.log(:info, "Approver email logic starting")
+  requester_email = requester_email_address
+  to = $evm.object['to_email_address']
+  from = $evm.object['from_email_address']
+  subject = "Request ID #{@miq_request.id} - Virtual machine request was denied"
```

Comment:
@tinaafitz 
Should this be service request was denied instead of Virtual Machine request was denied